data/sql/updates/pending_db_world/rev_1700314630271871000.sql

```diff
@@ -0,0 +1,83 @@
+-- Update remaining gameobjects 'Alliance Bonfire' with sniffed values
+DELETE FROM `gameobject` WHERE (`id` = 187922) AND (`guid` = 76308);
+DELETE FROM `gameobject` WHERE (`id` = 187924) AND (`guid` = 76335);
+DELETE FROM `gameobject` WHERE (`id` = 187925) AND (`guid` = 76311);
+DELETE FROM `gameobject` WHERE (`id` = 187926) AND (`guid` = 76303);
+DELETE FROM `gameobject` WHERE (`id` = 187927) AND (`guid` = 76330);
+DELETE FROM `gameobject` WHERE (`id` = 187928) AND (`guid` = 76307);
+DELETE FROM `gameobject` WHERE (`id` = 187929) AND (`guid` = 76329);
+DELETE FROM `gameobject` WHERE (`id` = 187930) AND (`guid` = 76352);
+DELETE FROM `gameobject` WHERE (`id` = 187935) AND (`guid` = 76345);
+DELETE FROM `gameobject` WHERE (`id` = 187938) AND (`guid` = 76319);
+DELETE FROM `gameobject` WHERE (`id` = 187939) AND (`guid` = 76321);
+DELETE FROM `gameobject` WHERE (`id` = 187940) AND (`guid` = 76313);
+DELETE FROM `gameobject` WHERE (`id` = 187941) AND (`guid` = 76354);
+DELETE FROM `gameobject` WHERE (`id` = 187942) AND (`guid` = 76359);
+DELETE FROM `gameobject` WHERE (`id` = 187943) AND (`guid` = 76326);
+DELETE FROM `gameobject` WHERE (`id` = 187944) AND (`guid` = 76301);
+DELETE FROM `gameobject` WHERE (`id` = 187945) AND (`guid` = 76323);
+DELETE FROM `gameobject` WHERE (`id` = 187946) AND (`guid` = 76342);
+DELETE FROM `gameobject` WHERE (`id` = 194032) AND (`guid` = 76368);
+DELETE FROM `gameobject` WHERE (`id` = 194035) AND (`guid` = 76372);
+DELETE FROM `gameobject` WHERE (`id` = 194036) AND (`guid` = 76367);
+DELETE FROM `gameobject` WHERE (`id` = 194038) AND (`guid` = 76362);
+DELETE FROM `gameobject` WHERE (`id` = 194040) AND (`guid` = 76365);
+DELETE FROM `gameobject` WHERE (`id` = 194044) AND (`guid` = 76376);
+DELETE FROM `gameobject` WHERE (`id` = 194045) AND (`guid` = 76370);
+DELETE FROM `gameobject` WHERE (`id` = 194049) AND (`guid` = 76375);
```

Comment:
this can be combined into one

Reply:
apart from that, thank you <3

Reply:
If each unique gameobject ID should match its unique guid this can't be combined, right?

Reply:
or you mean like delete from where (... and ...) or (... and ...)...

Reply:
ID IN (X) AND GUID IN (X).

They can be combined

Reply:
yes but then you would lose the 1-on-1 pairing of ID and GUID?

Reply:
I mean for this PR it's not really relevant just in general.